styles.css

```diff
@@ -0,0 +1,95 @@
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #fff;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+@media screen and (min-width: 375px) {
+  .fluidsConatiner {
+    width: 90%;
+  }
+
+  .navContainer {
+    background-color: #3c3a39;
+  }
+
+  .navbar {
+    height: 44px;
+
+    /* margin-top: 44px; */
+
+    /* width: 100wh; */
```

Comment:
- Please remove all commented out styles from your file to help make it easier to read and maintain 👍 👍